tests/integration/relations/pgbouncer_provider/test_pgbouncer_provider.py

```diff
@@ -362,30 +330,74 @@ async def test_legacy_relation_compatibility(ops_test: OpsTest):
         ops_test,
         unit_name=ops_test.model.applications[CLIENT_APP_NAME].units[0].name,
         relation_id=client_relation.id,
-        dbname=TEST_DBNAME,
+        dbname=APPLICATION_FIRST_DBNAME,
+        relation_name=FIRST_DATABASE_RELATION_NAME,
+    )
+
+
+@pytest.mark.client_relation
+async def test_multiple_pgb_can_connect_to_one_backend(ops_test: OpsTest, pgb_charm):
+    pgb_secondary = f"{PGB}-multi-backend"
```

Comment:
Maybe change `pgbouncer-k8s-multi-backend` to something like `pgbouncer-k8s-b` or even `secondary-pgbouncer-k8s` to avoid making people reading this code think that this PgBouncer application has multiple backends.